src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -107,10 +109,107 @@ public void Start()
                     UseShellExecute = false,
                     FileName = ExePath,
                     Arguments = GetCmdArgs(port),
-                    CreateNoWindow = true
+                    CreateNoWindow = true,
+                    WindowStyle = ProcessWindowStyle.Hidden,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true
+                }
+            };
+            process.OutputDataReceived += (sender, args) =>
+            {
+                string data = args.Data;
+                if (data != null)
+                {
+                    if (data.Contains("Server started"))
+                    {
+                        CreateChannelAndStremLogs();
+                    }
+                    WritelnToPane(data);
+                }
+            };
+            process.ErrorDataReceived += (sender, args) =>
+            {
+                string data = args.Data;
+                if (data != null)
+                {
+                    WritelnToPane(data);
                 }
             };
-            process.Start();
+            WritelnToPane($"Running {ExePath}");
+            try
+            {
+                process.Start();
+                process.BeginOutputReadLine();
+                process.BeginErrorReadLine();
+            }
+            catch (Exception e)
+            {
+                Debug.WriteLine("Unable to start SonarLint daemon: {0}", e);
+                WritelnToPane($"Unable to start SonarLint daemon {e.Message}");
+            }
+        }
+
+        private void CreateChannelAndStremLogs()
+        {
+            channel = new Channel($"{DAEMON_HOST}:{port}", ChannelCredentials.Insecure);
+            daemonClient = new StandaloneSonarLint.StandaloneSonarLintClient(channel);
+            ListenForLogs();
+            StartHeartBeat();
+        }
+
+        private async System.Threading.Tasks.Task ListenForLogs()
+        {
+            ISonarLintSettings settings = ServiceProvider.GlobalProvider.GetMefService<ISonarLintSettings>();
+            try
+            {
+                using (var streamLogs = daemonClient.StreamLogs(new Sonarlint.Void(), new CallOptions(null, null, channel.ShutdownToken).WithWaitForReady(true)))
+                {
+                    while (await streamLogs.ResponseStream.MoveNext())
+                    {
+                        var log = streamLogs.ResponseStream.Current;
+                        if (ShouldLog(settings, log))
+                        {
+                            WritelnToPane($"{log.Level} {log.Log}");
+                        }
+                    }
+                }
+            }
+            catch (RpcException e)
+            {
+                Debug.WriteLine("RPC failed: {0}", e);
+                WritelnToPane("RPC failed " + e);
+                throw;
+            }
+        }
+
+        private async System.Threading.Tasks.Task StartHeartBeat()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Change return type to 'void'; not a single caller uses the returned value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS3241)